qiime2/sdk/result.py

```diff
@@ -322,7 +354,7 @@ def validate(self, level='max'):
 
 
 class Visualization(Result):
-    extension = '.qzv'
+    extension = 'qzv'
```

Comment:
Same comment as above:

```suggestion
    extension = '.qzv'
```